### ISSUE TITLE
Support USB Connection to APC ES550?

### ISSUE THREAD

**Issue by whitedavidp** [norkator/apcupsd-monitor#113]
I was hoping for support of the standard USB connectivity of these UPS models. Thanks

**Comment by norkator** [comment]
> I was hoping for support of the standard USB connectivity of these UPS models. Thanks

You mean OTG like connection to phone? So usb cable is directly connected to phone.

These apps in the middle apcupsd is doing heavy lifting so doing this is most likely heavy job. I leave it to you or someone else who is interested to provide code.

**Comment by whitedavidp** [comment]
Yes, that is precisely what I meant. But in my case, it is an Android TV computer box on which I run all of my home automation stuff. It has multiple USB inputs. Thanks.
